packages/utils/upgrade/src/modules/version/range.ts

```diff
@@ -8,8 +8,11 @@ export const rangeFactory = (range: string): Version.Range => {
 };
 
 export const rangeFromReleaseType = (current: Version.SemVer, identifier: Version.ReleaseType) => {
-  const fromCurrentTo = (version: Version.LiteralVersion) => {
-    return rangeFactory(`>${current.raw} <=${version}`);
+  const fromCurrentTo = (version: Version.LiteralVersion, isUpperInclusive = true) => {
```

Comment:
Since the format isn't exactly the same everywhere, I feel like this helper introduces more complexity for a 3x one-liner, than it removes. What do you think about just using calls to `rangeFactory('>current < | <= version')`?

Reply:
(I know you just changed it already, sorry for coming in this late)